nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard
# Chaos Start / Stop with Flying Monkey and Toast

## Description
This code allows any user to click a button and allow the chaos monkey to run free and cause destruction.

## Related Issue
https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/issues/20
https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/issues/63
https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/issues/64
https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/issues/65

## Motivation and Context
The Chaos Monkey is a required feature for the project.

## How Can This Be Tested?
Click the button that says "Let me out to play!"
You should see the flying monkey fly by, and a toast pop-up at a set interval describing the destruction wrought by the Chaos Monkey.  
The button should change to "Put Ebenezer back in his Cage!"

## Notes
The Toast Interval is set at just 10 seconds for testing purposes. We can adjust the interval for production.

## Screenshots (if appropriate)
![Chaos](https://user-images.githubusercontent.com/51683901/111912494-dcf7e500-8a37-11eb-8742-6bf0a828b5ca.PNG)

## Types of changes
